lib/core/src/utils.rs

```diff
@@ -49,3 +52,38 @@ pub(crate) fn deserialize_tx_hex(tx_hex: &str) -> Result<Transaction> {
         |err| anyhow!("Could not deserialize transaction: {err:?}"),
     )?)?)
 }
+
+pub(crate) fn estimate_refund_fees(
+    swap: &SendSwap,
+    config: &Config,
+    output_address: &str,
+    is_lowball: bool,
+) -> Result<u64, PaymentError> {
+    let swap_script = swap.get_swap_script()?;
+    let electrum_config = ElectrumConfig::new(
+        config.network.into(),
+        &config.liquid_electrum_url,
+        true,
+        true,
+        100,
+    );
+    let swap_tx = boltz_client::LBtcSwapTxV2::new_refund(
+        swap_script,
+        &output_address.to_string(),
+        &electrum_config,
+        config.liquid_electrum_url.clone(),
+        swap.id.clone(),
+    )?;
+    let dummy_fees = Amount::from_sat(100);
+    let dummy_tx = swap_tx.sign_refund(&swap.get_refund_keypair()?, dummy_fees, None)?;
+
+    let fee_rate = if is_lowball {
+        config
+            .lowball_fee_rate()
```

Comment:
Yeah the `* 1000` should be removed so the absolute fee is calculated correctly

Reply:
No, we should instead make it obvious what unit we're using (e.g. rename `lowball_fee_rate` -> `lowball_fee_rate_msat_per_vbyte`).

LWK initializes the feerate to `100.0`: https://github.com/Blockstream/lwk/blob/ef7b0058439d496754f1d025f76c711baed8dc19/lwk_wollet/src/tx_builder.rs#L79 which is the 0.1 s/vB.

This means when we give a feerate arg to LWK, e.g. when building the LWK transactions, the our `LOWBALL_FEE_RATE_SAT_PER_VBYTE = 0.1` needs to be multiplied by 1000, which it is at the moment, to match the different unit expected by LWK.

Reply:
We could even have two feerate methods, for use internally (in s/vB) and for use with LWK (ms/vB).